packages/api-client/src/views/Request/RequestSection/RequestBody.vue

```diff
@@ -505,3 +505,8 @@ watch(
     </DataTable>
   </ViewLayoutCollapse>
 </template>
+<style scoped>
+:deep(.cm-content) {
+  font-size: var(--scalar-mini);
```

Comment:
@cameronrohani I think we could probably update this in the main code block component, the default font size there is hard coded and a little big in my opinion

Reply:
yes, it's actually used in the environment view, but since it's only used there we could switch the default!